doc/README.rst

```diff
@@ -3,9 +3,10 @@ Chapel Documentation
 
 This directory contains Chapel's documentation in two formats:
 
-``html/`` contains a web-rendered format of the documentation, which reflects
-the online documentation hosted at ``https://chapel-lang.org/docs``. This can be
-viewed by pointing a web browser of your choice to ``html/index.html``.
+``html/`` once generated by the ``make docs`` command, contains a web-rendered 
```

Comment:
I'd have this say "is generated by the ``make docs`` command and contains" instead